src/App.tsx

```diff
@@ -1,204 +1,69 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Todo } from './types/Todo';
+import { TodoError } from './types/TodoError';
+import { getTodos } from './api/todos';
+import { Filter } from './types/Filter';
+import { getPraperedTodos } from './services/todos';
+import { TodoList } from './components/TodoList';
+
+const USER_ID = 11968;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // #region STATE
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState<TodoError>(TodoError.NONE);
+  const [filter, setFilter] = useState<Filter>(Filter.ALL);
+  // #endregion
+
+  // #region EFFECT
+  useEffect(() => {
+    setIsLoading(true);
+
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => setErrorMsg(TodoError.LOAD))
+      .finally(() => setIsLoading(false));
+  }, [USER_ID]);
```

Comment:
unnecessary dependency: 'USER_ID', this value doesn't change, so it shouldn't be in dependencies array